content/en/quickstart/device-to-device.md

```diff
@@ -11,3 +11,24 @@ menu:
     weight: 20
 toc: true
 ---
+
+## Initialize CoAP(over UDP) Client
+
+## Discovery Cloud Server example 
```

Comment:
Write title as an action of what will a user learn.

Discover device on local network